entity-types/ext-dbt_job_run/golden_metrics.yml

```diff
@@ -0,0 +1,21 @@
+failed_models:
```

Comment:
Could you change the golen metric names to camelCase?

Reply:
Done. Thanks